addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -1063,6 +1337,37 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
     _getSelect: function () {
         return this.$target[0].querySelector('select');
     },
+    /**
+     * Copies all child elements of sourceFieldEl to destinationFieldEl
+     * 
+     * @private
+     * @param  {HTMLElement} sourceFieldEl
+     * @param  {HTMLElement} destinationFieldEl
+     */
+    _synchronizeFields(sourceFieldEl, destinationFieldEl) {
+        this._removeAllChild(destinationFieldEl);
+        for (const childrenEl of sourceFieldEl.children) {
+            destinationFieldEl.append(childrenEl.cloneNode(true));
+        }
+        // Regenerate IDs
+        for (const inputEl of destinationFieldEl.querySelectorAll('[id]:not(div)')) {
+            const newId = Math.random().toString(36).substring(2, 15);
+            const labelEl = destinationFieldEl.querySelector(`label[for="${inputEl.id}"]`);
+            if (labelEl) {
+                labelEl.setAttribute('for', newId);
+            }
+            inputEl.id = newId;
+        }
+        destinationFieldEl.setAttribute('data-type', sourceFieldEl.getAttribute('data-type'));
+    },
+    /**
+     * @param  {HTMLElement} el
+     */
+    _removeAllChild(el) {
+        while (el.firstChild) {
+            el.removeChild(el.lastChild);
+        }
+    }
```

Comment:
AllChildren would be more accurate english ;) But I would not create an utility function for a one line code:
```
[...el.childNodes].forEach(node => node.remove());
```
But you can keep it if you want but rename the method correctly